code/game/machinery/autolathe.dm

```diff
@@ -78,8 +67,9 @@
 	RefreshParts()
 
 /obj/machinery/autolathe/upgraded/gamma/Initialize()
-    . = ..()
-    adjust_hacked(TRUE)
+	. = ..()
+	files = new /datum/research/autolathe/gamma(src)
```

Comment:
Little bit out of scope but if a griefer wanted to, they could decon the autolathe and boom you can't craft buckshot anymore. Oh well.